build/vuejs-quiz-qpp/quiz.json

```diff
@@ -0,0 +1,258 @@
+[
+    {
+        "Question":
+          "In CSS, Flex is a shorthand property for which of the following properties?",
+        "Answer": "flex-grow, flex-shrink, and flex-basis",
+        "Distractor1": "justify-content, align-items, and align-content",
+        "Distractor2": "flex-direction, flex-wrap, and flex-flow",
+        "Distractor3": "flex-direction, order, and flex-flow",
+        "Explanation":
+          "Flex is a shorthand property for all three of these properties: flex-grow, flex-shrink, and flex-basis.",
+        "Link": "https://developer.mozilla.org/en-US/docs/Web/CSS/flex"
```

Comment:
In your original prototype [here](https://learnvue-quizapp.netlify.app/), there is a modal that displays the total number of questions correct when the user completes the quiz.
But there is no mention of explanations for each question or links to more detailed explanations.

If your revised prototype will include that, then it is fine to keep these fields here.
If not, then these explanation and link fields are not necessary.